doc/md_docs.go

```diff
@@ -102,9 +102,9 @@ func GenMarkdownCustom(cmd *cobra.Command, w io.Writer, linkHandler func(string)
 			cname := name + " " + child.Name()
 			link := cname + ".md"
 			link = strings.ReplaceAll(link, " ", "_")
-			buf.WriteString(fmt.Sprintf("* [%s](%s)\t - %s\n", cname, linkHandler(link), child.Short))
+			buf.WriteString(fmt.Sprintf("* [%s](%s) - %s\n", cname, linkHandler(link), child.Short))
 		}
-		buf.WriteString("\n")
+		// buf.WriteString("\n")
```

Comment:
Did you mean to comment this out? Or should it be removed entirely.

Reply:
> Did you mean to comment this out? Or should it be removed entirely.

I forgot to remove it. Thanks for reminding me. I've removed it and pushed the new version.